unable to build an image

```nix
{
  inputs = {
    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
    openwrt-imagebuilder.url = "github:astro/nix-openwrt-imagebuilder";
  };
  outputs =
    {
      self,
      nixpkgs,
      openwrt-imagebuilder,
      ...
    }@inputs:
    {
      packages.x86_64-linux.foo =
        let
          pkgs = nixpkgs.legacyPackages.x86_64-linux;

          profiles = openwrt-imagebuilder.lib.profiles { inherit pkgs; };

          config = profiles.identifyProfile "netgear_ex6100v2";
        in
        openwrt-imagebuilder.lib.build config;
    };
}
```

Got:

```
error: builder for '/nix/store/b5zcy8k3xhzzq9rh9nash5bqvqs8xlbk-openwrt-24.10.2-ipq40xx-generic-netgear_ex6100v2.drv' failed with exit code 2;
       last 25 log lines:
       > Directory table size 10286 bytes (10.04 Kbytes)
       >         44.72% of uncompressed directory table size (23001 bytes)
       > Number of duplicate files found 123
       > Number of inodes 1071
       > Number of files 783
       > Number of fragments 23
       > Number of symbolic links 203
       > Number of device nodes 1
       > Number of fifo nodes 0
       > Number of socket nodes 0
       > Number of directories 84
       > Number of hard-links 0
       > Number of ids (unique uids + gids) 1
       > Number of uids 1
       >      unknown (0)
       > Number of gids 1
       >    unknown (0)
       > 6508+1 records in
       > 6508+1 records out
       > 3332176 bytes (3.3 MB, 3.2 MiB) copied, 0.0119432 s, 279 MB/s
       > /nix/store/4bacfs7zrg714ffffbjp57nsvcz6zfkq-bash-5.3p3/bin/bash: ./netgear-fit-padding.py: /usr/bin/env: bad interpreter: No such file or directory
       > make[3]: *** [Makefile:22: /build/openwrt-imagebuilder-24.10.2-ipq40xx-generic.Linux-x86_64/build_dir/target-arm_cortex-a7+neon-vfpv4_musl_eabi/linux-ipq40xx_generic/tmp/openwrt-24.10.2-ipq40xx-generic-netgear_ex6100v2-squashfs-sysupgrade.bin] Error 126
       > make[2]: *** [Makefile:268: build_image] Error 2
       > make[1]: *** [Makefile:163: _call_image] Error 2
       > make: *** [Makefile:337: image] Error 2
       For full logs, run:
         nix log /nix/store/b5zcy8k3xhzzq9rh9nash5bqvqs8xlbk-openwrt-24.10.2-ipq40xx-generic-netgear_ex6100v2.drv
```

https://github.com/openwrt/openwrt/blob/v24.10.2/target/linux/ipq40xx/image/netgear-fit-padding.py#L1

I'm not sure, but I think this builder already includes a python3:

https://github.com/astro/nix-openwrt-imagebuilder/blob/69d1477277572efcfe33763c3e25ab0a1605355a/builder.nix#L148-L152

Any idea?

--- a/builder.nix
+++ b/builder.nix
@@ -100,7 +100,7 @@ pkgs.stdenv.mkDerivation ({
       else throw "No valid image builder found!";
 
   postPatch = with pkgs; ''
-    patchShebangs scripts staging_dir/host/bin
+    patchShebangs scripts staging_dir/host/bin target/linux
     substituteInPlace rules.mk \
       --replace "SHELL:=/usr/bin/env bash" "SHELL:=${runtimeShell}" \
       --replace "/usr/bin/env true" "${coreutils}/bin/true" \

